samples/bluetooth/periodic_sync/Kconfig

```diff
@@ -0,0 +1,15 @@
+#
+# Copyright (c) 2024 Hang Fan
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+mainmenu "Bluetooth: Periodic Advertising Sync"
+
+config PER_ADV_NAME
+	string "Target Periodic Advertising Name"
+	default "Test Periodic Advertising"
+	help
+	  Name of target advertising for Periodic Advertising Synchronization.
```

Comment:
IMO this should be optional - I'd set the default value to `""` and only if it is not `""` I'd do a name comparison, else just sync to the first device (same behavior as before). 

This way users can choose specific advertiser, or just use with whatever is first.